content/en/tracing/trace_collection/custom_instrumentation/rust.md

```diff
@@ -10,12 +13,219 @@ further_reading:
       text: 'Explore your services, resources, and traces'
 ---
 
-Datadog supports custom instrumentation for Rust applications when you use the [OpenTelemetry SDK][1].
-For more information on setting this up, see [How to Monitor Your Rust Applications with OpenTelemetry][2] on the blog.
+{{< callout header="false" btn_hidden="true"  >}}
+  The Datadog Rust SDK is in Preview.
+{{< /callout >}}
+
+Datadog provides support for custom instrumentation in Rust applications through the [`datadog-opentelemetry` crate][3]. This library is built on the OpenTelemetry (OTel) API and SDK, providing a tracer that includes Datadog-specific features and an exporter.
+
+Because this library is built on OpenTelemetry, you use the standard OpenTelemetry API to create traces and spans.
+
+## Setup
+
+To configure your Rust application to send OpenTelemetry traces to Datadog, you need to add the correct dependencies and initialize the tracer provider with the Datadog exporter.
+
+### 1. Add dependencies
+
+Add `datadog-opentelemetry` and the core `opentelemetry` crate to your `Cargo.toml`.
+
+```toml
+[dependencies]
+# The main Datadog OTel library
+datadog-opentelemetry = { version = "0.1" } # Note: Check crates.io for latest
```

Comment:
Going to bump it one time before reinvent, so it's likely going to end up being 0.2